microbench/pom.xml

```diff
@@ -73,14 +73,19 @@
     <dependency>
       <groupId>org.openjdk.jmh</groupId>
       <artifactId>jmh-core</artifactId>
-      <version>0.9</version>
+      <version>1.7.1</version>
```

Comment:
Been a while since we updated I guess...do your features depend upon this update though? I'm not against is...just curious.